frontend/lib/src/components/widgets/DataFrame/columns/ChartColumn.ts

```diff
@@ -193,7 +187,7 @@ function BaseChartColumn(
  * This column type is currently read-only.
  */
 export function LineChartColumn(props: BaseColumnProps): BaseColumn {
-  return BaseChartColumn("line_chart", props, "line")
+  return BaseChartColumn("line_chart", props, "area")
```

Comment:
This seems odd that it's a BaseChartColumn but its chart type is "area". I understand glide-data-grid is adding a new area chart but maybe I'm missing something?

Reply:
Do we need a new AreaChartColumn?

Reply:
What is called `area` now in GDG, is the old line chart. In a subsequent PR, we will eventually change this to `line` and introduce a dedicated `AreaChartColumn`.

Reply:
I added a comment 👍